scripts/tsconfig.json

```diff
@@ -6,20 +6,20 @@
     "forceConsistentCasingInFileNames": true,
     "lib": [
       "dom",
-      "es2019",
+      "es2020"
     ],
-    "module": "commonjs",
-    "moduleResolution": "node",
+    "module": "CommonJS",
+    "moduleResolution": "nodenext",
     "outDir": "build/",
     "pretty": true,
-    "target": "es2017",
```

Comment:
For my own edification, do we need to change all of these values? Or are some remnants of tweaking the build to get `execa` working?

Reply:
I think this was the sort of minimum set of changes I could use to get it working, I could do a little remove and replace to see if it will let us change fewer optinos

Reply:
Ok so I think the only change which is truly necessary is changing `moduleResolution` to `nodenext` - without that we have no dynamic import. any reason not to update to `ES2020` while we're in here?

Reply:
I think the only change we truly need is the `moduleResolution: "nodenext"`. without it typescript will compile `import("execa")` to `require("execa")`

Reply:
ahh sorry for the noise, for some reason the 'single comment' vs 'review comment' thing I always click the wrong button (I think I deleted one by accident for good measure)

Reply:
I think that's OK (to bump the others as well) - probably good to demarcate the ones we need for this vs the ones that are "while we're in here" in the git commit msg, just in case we're `git blame`ing this file and are like "what did you see @alicewriteswrongs, what did you seeeeee?!?!"

Reply:
haha ok sure, that makes sense to me

Reply:
I just rewrote my commit on the branch to include a call-out about that